{
  "comments": [
    {
      "key": {
        "uuid": "6ec1af18_276f9c27",
        "filename": "/COMMIT_MSG",
        "patchSetId": 8
      },
      "lineNbr": 9,
      "author": {
        "id": 366
      },
      "writtenOn": "2015-09-08T01:43:10Z",
      "side": 1,
      "message": "Somewhat? ;-)",
      "revId": "25b9ea5d9b21a9d3adc0e1a7cafd3ff35687fc83",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "6ec1af18_674a8485",
        "filename": "/COMMIT_MSG",
        "patchSetId": 8
      },
      "lineNbr": 9,
      "author": {
        "id": 98
      },
      "writtenOn": "2015-09-08T17:19:51Z",
      "side": 1,
      "message": "I basically checked init, ready, saveIntent, saveAttempt, and saveSuccess. Need to check abandon (does it cover all cases?) and saveFailure, and check everything validates.",
      "parentUuid": "6ec1af18_276f9c27",
      "revId": "25b9ea5d9b21a9d3adc0e1a7cafd3ff35687fc83",
      "serverId": "e9e9afe9-4712-486d-8885-f54b72dd1951",
      "unresolved": false
    }
  ]
}